src/App/pages/HomePage/components/CategoryTabs/components/Tab/Tab.tsx

```diff
@@ -0,0 +1,24 @@
+import Button, { ButtonProps } from 'components/Button';
+import { TNamedCategory } from 'store/types';
+import styles from './Tab.module.scss';
+
+export type TabProps = {
+  category: TNamedCategory;
+  isActive?: boolean;
+} & ButtonProps;
+
+const Tab: React.FC<TabProps> = (props) => {
+  const { category, isActive, ...rest } = props;
+
+  return (
+    <Button
+      className={styles['TabButton']}
+      text={category.name}
+      variant={isActive ? 'solid' : 'soft'}
+      key={category.category}
+      {...rest}
+    />
```

Comment:
можно добавить коллбэк `onClick: (category: TNamedCategory) => void` и тут обрабатывать клик
```suggestion
  const handleClick = React.useCallback(() => onClick(category), [category]);

  return (
    <Button
      {...rest}
      className={styles['TabButton']}
      text={category.name}
      variant={isActive ? 'solid' : 'soft'}
      key={category.category}
      onClick={handleClick}
    />
```